core/mock/databaseWriterMock.go

```diff
@@ -0,0 +1,31 @@
+package mock
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/elastic/go-elasticsearch/v7/esapi"
+)
+
+type DatabaseWriterMock struct {
```

Comment:
add comments to this stub/mock. I think this resembles more to a stub rather than mock

Reply:
done.